pysparkling/sql/types.py

```diff
@@ -1159,20 +1133,20 @@ def convert_dict(obj):
 
 _acceptable_types = {
     BooleanType: (bool,),
-    ByteType: (int, long),
-    ShortType: (int, long),
-    IntegerType: (int, long),
-    LongType: (int, long),
+    ByteType: (int,),
+    ShortType: (int,),
+    IntegerType: (int,),
+    LongType: (int,),
     FloatType: (float,),
     DoubleType: (float,),
     DecimalType: (decimal.Decimal,),
-    StringType: (str, unicode),
+    StringType: (str,),
     BinaryType: (bytearray,),
-    DateType: (datetime.date, datetime.datetime),
+    DateType: (datetime.date, datetime.datetime,),
```

Comment:
can you revert adding commas to tuples with more than one element?

Reply:
Ok, that's easy to do... I just think it is nicer that all are looking equal. I know it doesn't matter, but I like uniformity.

Reply:
I disagree.